java-frontend/src/test/files/se/customresources/FactoryClosed.java

```diff
@@ -4,8 +4,9 @@
 //    factoryMethod = org.sonar.custom.ResourceFactory#createResource(java.lang.String)
 //    closingMethod = org.sonar.custom.GenericResource#closeResource(java.lang.String)
 
+interface OpenedResource {}
 
-public class GenericResource {
+public class GenericResource implements OpenedResource {
```

Comment:
Why do you need this change?! if it's only to get a correct resource name, it's wrong.